app/services/fact_parser.rb

```diff
@@ -79,10 +79,10 @@ def interfaces
   end
 
   # tries to detect primary interface among interfaces using host name
-  def suggested_primary_interface(host)
+  def suggested_primary_interface(hname)
```

Comment:
Nit: please use `hostname` here, nobody will know what `hname` means.

In addition, this breaks the API which a lot of plugins use. :-(